src/main/content/_assets/css/guides.css

```diff
@@ -10,7 +10,6 @@
  *******************************************************************************/
 
 /* INTRO */
-
 #guides_intro_background {
```

Comment:
We should delete the styling for `#guides_intro_background` and `#intro_logo`, if we no longer have the HTML.

Reply:
Done